packages/react-next/src/components/TextField/MaskedTextField/MaskedTextField.tsx

```diff
@@ -15,385 +14,374 @@ import {
   insertString,
   parseMask,
 } from './inputMask';
+import { useConst } from '@uifabric/react-hooks';
+
+interface IMaskedTextFieldState {
+  maskCharData: IMaskValue[];
+  isFocused: boolean;
+  moveCursorOnMouseUp: boolean;
+  changeSelectionData: {
+    changeType: 'default' | 'backspace' | 'delete' | 'textPasted';
+    selectionStart: number;
+    selectionEnd: number;
+  } | null;
+}
 
-/**
- * State for the MaskedTextField component.
- */
-export interface IMaskedTextFieldState {
+interface IMaskedTextField extends ITextField {
```

Comment:
This should go in `TextField.types.ts` (and add a comment with `{@docCategory TextField}`). Also realized you'll need to add `componentRef?: IRefObject<IMaskedTextField>` to `IMaskedTextFieldProps`.